src/pages/question.js

```diff
@@ -22,14 +44,15 @@ function Question() {
   const [content, setContent] = useState("");
   const [questionId, setQuestionId] = useState(0);
   const [answers, setAnswers] = useState([]);
+  const [answered, setAnswered] = useState(false);
```

Comment:
You can do this in FE, but this sort of business logic should be in the BE as well. Otherwise a user can call your API directly and go around this rule.

Reply:
Yea, I guess I should make the pair (user,question) of answers unique on the BE and that should solve it